### DISCUSSION TITLE
Feature request: support attaching more types of files

### DISCUSSION THREAD

**Discussion by aviflax** [community/community#4077] [Discussions] [answered]
I wanted to share a PDF with my team just now, but I ended up having to upload it into our Google Drive shared drive first, and configure the sharing settings (which has high cognitive load) and then get a link, and add it to my post to our team discussion.

Hence this feature request: please consider supporting the direct attachment of PDF, TXT, and MD files to team discussion threads/posts.

(Archive files (zip, tgz, etc) would be cool too but I would guess there might be greater security concerns with those.)

Thank you!

**Comment by xmo-odoo** [comment, marked as answer]
> Archive files (zip, tgz, etc) would be cool too but I would guess there might be greater security concerns with those.

`.zip` and `.tar.gz` (though not `.tgz`) are already supported, though more modern algorithms would not be remiss e.g. `.zst[d]` for the ability to have both very high compression throughputs and very high compression ratios (with very high decompression throughputs either way), and `.xz` for extremely high compression ratios (but at a steep throughput cost in both compression and decompression).

An other thing which would be very useful would be supporting [`.age`](https://github.com/FiloSottile/age) files: the CLI utility works well with github (as it supports ssh keys which github exposes), meaning it can be used to compensate for the lack of private issues / attachments.

**Reply by aviflax** [reply]
Testing!

[message.txt.zip](https://github.com/github/feedback/files/7540430/message.txt.zip)
[message.pdf](https://github.com/github/feedback/files/7540439/message.pdf)

Looks like you’re right.

(I’m a little baffled as to how/why I was under the impression that such files weren’t supported. Maybe it was just an intermittent error/bug 🤷. Oh well.)

Thank you!
